### DISCUSSION TITLE
Change order of columns in ERD?

### DISCUSSION THREAD

**Discussion by torstenberg** [mermaid-js/mermaid#3717] [Q&A]
Hi,

is it possible to change the order of the column displayed for a table in an ERD diagram? Currently, the column type comes first, then the column name. In most implementations, I see the column name first and I would like to have the diagram do that. However, the documentation does not state whether this is possible.

Any idea?

**Comment by weedySeaDragon** [comment]
The order is hard coded, so can't be changed by a user.
